core/src/main/java/bisq/core/account/witness/AccountAgeWitnessService.java

```diff
@@ -886,4 +887,24 @@ public String signInfoFromAccount(PaymentAccount paymentAccount) {
         var witness = getMyWitness(paymentAccount.getPaymentAccountPayload());
         return Utilities.bytesAsHexString(witness.getHash()) + "," + Utilities.bytesAsHexString(pubKey.getEncoded());
     }
+
+    public Tuple2<AccountAgeWitness, byte[]> signInfoFromString(String signInfo) {
+        var parts = signInfo.split(",");
```

Comment:
Ah now I see why u use "," for concatination....
A pipe or a static field for it would make it a bit more intuitive... but no worry... being picky ;-)

Reply:
Won't change, just easier to read with `,` as separator.